### PR TITLE
docs: document SYSTEMD_REPART_MKFS_* envvar in repart.d man page

### PR DESCRIPTION
**Pull request by gdonval**

### CODE REVIEW

**Comment by gdonval** [comment]
Not sure what to do with `mkfs.FILESYSTEM`.

File: man/repart.d.xml [hunk 1]
```diff
@@ -852,6 +852,19 @@
...
+  <refsect1>
+    <title>Environment Variables</title>
+    
+    <para>Extra filesystem formatting options can be provided using filesystem-specific environment variables:
+    <varname>$SYSTEMD_REPART_MKFS_OPTIONS_BTRFS</varname>, <varname>$SYSTEMD_REPART_MKFS_OPTIONS_XFS</varname>,
+    <varname>$SYSTEMD_REPART_MKFS_OPTIONS_VFAT</varname>, <varname>$SYSTEMD_REPART_MKFS_OPTIONS_EROFS</varname>,
+    and <varname>$SYSTEMD_REPART_MKFS_OPTIONS_SQUASHFS</varname>. Each variable accepts valid
+    <command>mkfs.FILESYSTEM</command> command-line arguments. The content of those variables is currently
+    unchecked.
+    </para>
```
Comment (daandemeyer) [inline man/repart.d.xml]: Please format the same as other `Environment` sections. Grep for `<title>Environment</title>` for examples

**Review by DaanDeMeyer — CHANGES_REQUESTED** [review]

File: man/repart.d.xml (hunk 1 above)
Comment (gdonval) [reply man/repart.d.xml]: Do you want a full exhaustive list with each filesystem explicitly named? Or something like in `bootctl` would work?

Comment (keszybz) [reply man/repart.d.xml]: The general format is fine, but please use
`<command>mkfs.<replaceable>filesystem</replaceable></command>` so this gets formatted.

In the last sentence, maybe instead say "The contents of those variables are passed as-is to the command, without any verificiation."

**Pull request merged**